tests/integration/StorageWriter/IPFS.test.ts

```diff
@@ -0,0 +1,90 @@
+import { describe } from 'riteway'
+import { is } from 'ramda'
+import fetch from 'node-fetch'
+
+import { IPFS } from '../../../src/StorageWriter/IPFS'
+import { offendingClaim, workingClaim, offendingClaimWithInvlaidUtf8CharsRemoved } from './claimData'
+
+const isString = is(String)
+
+const IPFS_URL = 'http://localhost:5001'
```

Comment:
Can you add the possibility to set an env var as option?

Reply:
It was on my todo list :) Sorry meant to add 'dont merge' and 'work in progress'